packages/web3-utils/README.md

```diff
@@ -39,12 +41,20 @@ console.log(Utils);
 }
 ```
 
+Import what you need
```

Comment:
"Import several or just one method"

Reply:
cool will do this once i've commented and got replies from your other comments (so they don't hide) 😄